ccm_web/client/src/pages/BulkSectionCreate.tsx

```diff
@@ -265,59 +263,15 @@ function BulkSectionCreate (props: BulkSectionCreateProps): JSX.Element {
     })
   }
 
-  const open = Boolean(anchorEl)
-
-  const handlePopoverOpen = (event: React.MouseEvent<HTMLElement, MouseEvent>): void => {
-    setAnchorEl(event.currentTarget)
-  }
-
-  const handlePopoverClose = (): void => {
-    setAnchorEl(null)
-  }
-
   const renderUploadHeader = (): JSX.Element => {
+    const fileData =
+`SECTION_NAME
+Section 001`
```

Comment:
This is fine, But I feel that all the sample data that goes in the file download coming from a separate file that will keep things a bit organized. This sample download concept also shared across other features. So you will have one file to that serves up the content. It can be event class

```
class SampleDowloadData {
	  construtor ( feature :string ) {
	  this.feature= feature
	  }
	  sendSampleData(){
	  if(feature == 'createSection`){
	  const fileData =`SECTION_NAMESection 001`
	  return fileData
	  }
}

Call it like `new SampleDowloadData(sectionCreate)`
```

It might feel bit complicated but separating this as a separate file will keep things neat.

Reply:
@pushyamig I added a component